cds/config.py

```diff
@@ -26,23 +26,56 @@
 
 from __future__ import absolute_import, print_function
 
+import os
+
 
 # Identity function for string extraction
 def _(x):
     """Indentity function."""
     return x
 
+# Database
+SQLALCHEMY_DATABASE_URI = os.environ.get(
+    "SQLALCHEMY_DATABASE_URI",
+    "postgresql+psycopg2://localhost/cds")
+SQLALCHEMY_ECHO = False
+
 # Default language and timezone
 BABEL_DEFAULT_LANGUAGE = 'en'
 BABEL_DEFAULT_TIMEZONE = 'Europe/Zurich'
-I18N_LANGUAGES = [
-]
+I18N_LANGUAGES = []
+
+# Distributed task queue
+BROKER_URL = "redis://localhost:6379/0"
```

Comment:
Should also use the environment variable as on line 38.
With 'APP_BROKER_URL' or  'APP_CACHE_REDIS_URL' as specified in the docker-compose.yml